### DISCUSSION TITLE
It keeps seeing vm as an error

### DISCUSSION THREAD

**Discussion by Joel-eth** [Cyfrin/foundry-full-course-cu#1646] [General]
The compiler keeps seeing vm as an error when I run and compile scripts. Has the vm cheatcode been deprecated or are there any other alternatives?

**Comment by EngrPips** [comment]
Please provide details so that we can have more context to help you. Show the script you are trying to run that result to getting a `vm error`

**Comment by Joel-eth** [comment]
sorry for not giving a proper description
![● DeploySimpleStorage s sol - hello_foundry  WSL_ Ubuntu  - Visual Studio Code 4_26_2024 3_21_13 PM](https://github.com/Cyfrin/foundry-full-course-f23/assets/113260217/91245496-8783-4841-9edc-834fc48214b1)
![● DeploySimpleStorage s sol - hello_foundry  WSL_ Ubuntu  - Visual Studio Code 4_26_2024 3_53_09 PM](https://github.com/Cyfrin/foundry-full-course-f23/assets/113260217/e1398b9a-1136-4298-b0b2-03e86ce7ab3c)

**Reply by EngrPips** [reply]
Please put `is Test` at the front of DeploySimpleStorage, as I show below.

```javascript
contract DeploySimpleStorage is Test {
}
```
